content/blog/2018/data-plane-setup/index.md

```diff
@@ -0,0 +1,533 @@
+---
+title: Demystifying Istio's Sidecar Injection Model 
+description: De-mystify how Istio manages to plugin its data-plane components into an existing deployment. Deep-dive into sidecar injection, how it is done whether manually or `namespaceSelector` based automatic injection. And also how the iptables are modified in the pod namespace to redirect the traffic to the injected proxy. 
+publishdate: 2018-12-12
+subtitle:
+attribution: Manish CHUGTU
+twitter: chugtum 
+weight: 78
```

Comment:
This number will need to be adjusted before publishing to ensure proper ordering on the site.

Reply:
Got that - I am leaving it like that as of now.